web/docs/use/introspect/README.md

```diff
@@ -1,5 +1,233 @@
+[DuckDB](https://duckdb.org/docs/guides/meta/list_tables), the `show` equivalent
+would be `SHOW TABLES` or `DESCRIBE`.
+
+You can invoke the `show` command with three positional arguments:
+
+1. `vast show concepts`
+2. `vast show models`
+3. `vast show schemas`
+
+Options (1) and (2) show taxonomy details about concepts and models, and (3)
+displays all known types, both from statically specified schemas in
+configuration files as well as dynamically generated schemas at runtime.
+
+Example output of (3) may look as follows:
+
+```bash
+vast show schemas
+```
+
+```json
+[
+  {
+    "suricata.flow": {
+      "record": {
+        "timestamp": {
+          "timestamp": "time"
+        },
+        "flow_id": {
+          "type": "count",
+          "attributes": {
+            "index": "hash"
+          }
+        },
+        "pcap_cnt": "count",
+        "vlan": {
+          "list": "count"
+        },
+        "in_iface": "string",
+        "src_ip": "addr",
+        "src_port": {
+          "port": "count"
+        },
+        "dest_ip": "addr",
+        "dest_port": {
+          "port": "count"
+        },
+        "proto": "string",
+        "event_type": "string",
+        "community_id": {
+          "type": "string",
+          "attributes": {
+            "index": "hash"
+          }
+        },
+        "flow": {
+          "suricata.component.flow": {
+            "record": {
+              "pkts_toserver": "count",
+              "pkts_toclient": "count",
+              "bytes_toserver": "count",
+              "bytes_toclient": "count",
+              "start": "time",
+              "end": "time",
+              "age": "count",
+              "state": "string",
+              "reason": "string",
+              "alerted": "bool"
+            }
+          }
+        },
+        "app_proto": "string"
+      }
+    }
+  },
+  {
+    "suricata.stats": {
+      "record": {
+        "timestamp": {
+          "timestamp": "time"
+        }
+      }
+    }
+  }
+]
+```
+  
+You can also render the output in YAML:
+
+```bash
+vast show schemas --yaml
+```
+
+```yaml
+- suricata.flow:
+    record:
+      timestamp:
+        timestamp: time
+      flow_id:
+        type: count
+        attributes:
+          index: hash
+      pcap_cnt: count
+      vlan:
+        list: count
+      in_iface: string
+      src_ip: addr
+      src_port:
+        port: count
+      dest_ip: addr
+      dest_port:
+        port: count
+      proto: string
+      event_type: string
+      community_id:
+        type: string
+        attributes:
+          index: hash
+      flow:
+        suricata.component.flow:
+          record:
+            pkts_toserver: count
+            pkts_toclient: count
+            bytes_toserver: count
+            bytes_toclient: count
+            start: time
+            end: time
+            age: count
+            state: string
+            reason: string
+            alerted: bool
```

Comment:
@dominiklohmann Looking at the output, I realized that this doesn't conform to the spec: record fields are ordered, so the `record` object must map to an array. From our [YAML Types](https://docs.google.com/document/d/1thYBUQ2mbobUyLbLF8Rc0LkexKK7UrnVuACiuCoYXXU/edit):

```yaml
connection:
  record:
  - src_ip: addr
  - dst_ip: addr
  - src_port: port
  - dst_port: port
  - proto: string
```

Reply:
I fixed this in the most recent commit. I did not touch your docs, but rather only the code and the test baselines.

Reply:
I'll adapt the docs. Now that I can just get the Docker container from GHCR, updating will be a breeze.